Update virtual-machines-common-prepay-reserved-vm-instances.md

jboeshart opened the pull request:
Updating the link to the Mange Azure Reserved Virtual Machine Instances document, as it only linked to the main RI product page.

azurecla commented:
Hi __@thomasberman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

BryanLa commented:
Hi @cabailey - would you be able to review this suggested change for us? If it passes your review, we will likely need to have it resubmitted, as GitHub is predicting a merge conflict (the file has probably changed since this suggestion was made). Thanks!

cabailey commented:
@thomasberman - That's a very useful addition, but we would need to make an equivalent change for the CLI tutorial (https://azure.microsoft.com/en-us/documentation/articles/key-vault-manage-with-cli/), and the equivalent command here would be:

azure keyvault set-policy -–vault-name 'ContosoKeyVault' --spn 8f8c4bbd-485b-45fd-98f7-ec6300b7b4ed --perm-to-secrets '["Get"]'

If you're able to make this change as well - great!  If not, we'll include it next time we update this article.

BryanLa commented:
@thomasberman - also, this particular article has undergone additional updates, between the time you submitted your suggestions and now, so we are unable to merge this PR due to merge conflicts. Feel free to submit both your original suggestion to key-vault-get-started.md in a new PR, as well as your same suggestion to key-vault-manage-with-cli.md, and I can merge both once you're ready.

Thanks very much for your contributions!

BryanLa commented:
Closing per above, due to merge conflicts.

thomasberman commented:
Great, I will make the change to the CLI article, and submit a new PR for this one! Thanks.

jboeshart committed:
Update virtual-machines-common-prepay-reserved-vm-instances.md

Updating the link to the Mange Azure Reserved Virtual Machine Instances document, as it only linked to the main RI product page.
